presets/change-history/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export * as AttributeDefinition from '@commercetools-test-data/attribute-definition';
+export * as AttributeType from '@commercetools-test-data/attribute-type';
+export * as Category from '@commercetools-test-data/category';
+export * as Channel from '@commercetools-test-data/channel';
+export * as Commons from '@commercetools-test-data/commons';
+export * as Core from '@commercetools-test-data/core';
```

Comment:
Not sure if I would include these packages in this preset. In particular `core`, I don't think it should belong in here.